test/dust-test.js

```diff
@@ -18,10 +19,13 @@ describe("bundalo dust bundler @dust@", function () {
 			if (err) {
 				return done(err);
 			}
-			if (data.greeting && bundloo.__cache()[path.normalize('nest/dusta.properties')] && !bundlee.__cache()[path.normalize('nest/dusta.properties')]) {
+			try {
+				assert.ok(data.greeting);
+				assert.ok(bundloo.__cache()[path.normalize('nest/dusta.properties')]);
+				assert.ok(!bundlee.__cache()[path.normalize('nest/dusta.properties')]);
```

Comment:
Ugh. Learning on a Friday... Ok I know what a domain is now :) So does mocha explicitly make use of these?